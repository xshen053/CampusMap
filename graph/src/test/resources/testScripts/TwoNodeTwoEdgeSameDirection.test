# Tests adding two nodes, one edge between two nodes
# and one points to itself

# Create a graph
CreateGraph TNOESD_graph

# Add two nodes
AddNode TNOESD_graph a
AddNode TNOESD_graph b

# Add one edge
AddEdge TNOESD_graph a a AA
AddEdge TNOESD_graph a b AB

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
ListNodes TNOESD_graph
ListChildren TNOESD_graph a
ListChildren TNOESD_graph b