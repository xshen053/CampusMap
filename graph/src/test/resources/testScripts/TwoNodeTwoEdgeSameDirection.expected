# Tests adding two nodes, one edge between two nodes
# and one points to itself

# Create a graph
created graph TNOESD_graph

# Add two nodes
added node a to TNOESD_graph
added node b to TNOESD_graph

# Add one edge
added edge AA from a to a in TNOESD_graph
added edge AB from a to b in TNOESD_graph

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
TNOESD_graph contains: a b
the children of a in TNOESD_graph are: a(AA) b(AB)
the children of b in TNOESD_graph are: